src/ts/uimanager.ts

```diff
@@ -10,6 +10,22 @@ import {BrowserUtils} from './browserutils';
 import { TimelineMarker, UIConfig } from './uiconfig';
 import { PlayerAPI, PlayerEventCallback, PlayerEventBase, PlayerEvent, AdEvent, LinearAd } from 'bitmovin-player';
 import { VolumeController } from './volumecontroller';
+import { i18n, CustomVocabulary, Vocabularies } from './localization/i18n';
+
+export interface LocalizationConfig {
+  /**
+   * defines the preferred locale
+   */
+  language?: 'en' | 'de' | string;
+  /**
+   * (default: true) will  enable/disable auto-detection and selection of browsers preferred language
+   */
+  browserLanguageDetection?: boolean;
```

Comment:
I think these two should be exclusive. If I set `language: 'de'`, I want my UI to be in German, the additional need to set `browserLanguageDetection: false` seems like it should not be necessary - also when I choose German, it would be unexpected if the language detection chooses another language instead.

Maybe this could be solved by removing `browserLanguageDetection` and adding an 'auto' option to `language`?

Reply:
Better yet, we introduce a `language: 'auto'` setting which is also the default, i.e. if the `language` property is missing in the config, `'auto'` will be set as default.